Cargo.toml

```diff
@@ -34,7 +35,9 @@ read_core = []
 # Read support for all file formats (including unaligned files).
 read = ["read_core", "archive", "coff", "elf", "macho", "pe", "unaligned"]
 # Core write support. You will need to enable some file formats too.
-write_core = ["crc32fast", "indexmap/std", "std"]
+write_core = ["crc32fast", "indexmap", "hashbrown"]
+# Core write support with libstd features. You will need to enable some file formats too.
+write_std = ["write_core", "std", "indexmap/std", "crc32fast/std"]
```

Comment:
This now needs to be included in the `all` and `doc` features somehow. I'm inclined to do this by changing `write` to depend on `write_std` instead of `write_core`, since then the meaning of `write` is unchanged by this PR.

Reply:
Agreed on both counts! I can make those changes shortly; in fact, my latest commit was merely an initial proposal for changes to the cargo features in order to allow you guys to check it out first.

We can definitely make `write` activate `std` too.